SDPi_Supplement/asciidoc/volume1/tf1-main.adoc

```diff
@@ -2,3 +2,37 @@
 
 NOTE:  This document integrates all the content that makes up the supplement volume 1 document.
 
+IMPORTANT:  Migrate the content below from main.adoc into this document OR rename main.adoc to this ... though the content is different.  +
+{empty} +
+*_See the Word document for what this is supposed to look like_*.
+
+
+[#clause-profiles]
```

Comment:
Another danling block that - I suppose - is intended to be merged with the subsequent one:

```asciidoc
[#vol1_clause_profiles,sdpi_offset=1]
== Profiles
```

Reply:
File updated as requested